subsys/bluetooth/services/fast_pair/Kconfig.fast_pair

```diff
@@ -184,6 +186,36 @@ config BT_FAST_PAIR_STORAGE_INTEGRATION_INIT_PRIORITY
 
 endif # BT_FAST_PAIR_STORAGE_INTEGRATION
 
+config BT_FAST_PAIR_BOND_MANAGER
```

Comment:
Please add it as a `select` or `imply` option to the use case layer for the input device and mouse use case.